sway-lib-std/src/tx.sw

```diff
@@ -200,8 +217,14 @@ pub fn tx_input_coin_owner(input_ptr: u32) -> Address {
         mcpi buffer ptr i32;
         // `buffer` now points to the 32 bytes
         buffer: b256
-    });
-
+    }
+}
+/// If the input's type is `InputCoin`, return the owner.
+/// Otherwise, undefined behavior.
+pub fn tx_input_coin_owner(index: u64) -> Address {
+    let input_ptr = tx_input_pointer(index);
+    // Need to skip over six words, so offset is 8*6=48
+    let owner_addr = ~Address::from(b256_from_pointer_offset(input_ptr, 48));
     owner_addr
```

Comment:
```suggestion
    ~Address::from(b256_from_pointer_offset(input_ptr, 48))
```

Reply:
Resolved in [a7c7a91](https://github.com/FuelLabs/sway/pull/2142/commits/a7c7a91776162bd5d4a0bbbbf94c14c066c83e9a)